engines/bastion_katello/app/assets/javascripts/bastion_katello/packages/details/package.controller.js

```diff
@@ -24,6 +25,7 @@ angular.module('Bastion.packages').controller('PackageController',
         }
 
         $scope.installedPackageCount = undefined;
+        $scope.newHostDetailsUI = newHostDetailsUI;
```

Comment:
This is true/false correctly when debugging

Reply:
Ensure that every page that uses this new variable with an ng-if gets the string to boolean fix.